assets/src/js/country_selector.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * The script basically add/remove the tabindex attribute within the country selector
+ * It helps to improve SEO
```

Comment:
I think the script description is not accurate enough (for instance, the script does nothing related to `tabindex`).
IMO, it would be important to mention why we are switching positions between the footer elements, otherwise, the purpose of this code may be hard to understand in the future.

Reply:
Oh sorry 🙏 ! I've noticed this after I merge the PR. Here is another #2456  which follows your suggestion.